.circleci/config.yml

```
@@ -50,8 +50,8 @@ jobs:
       - setup_remote_docker
       - run: docker pull gcr.io/projectriff/streaming-http-adapter:0.1.3
       - run: docker pull gcr.io/projectriff/node-function:0.6.1
-      - run: docker load < /tmp/workspace/pack-18-build.tar
-      - run: docker load < /tmp/workspace/pack-18-run.tar
+      - run: docker load < $(ls /tmp/workspace/pack-*-build.tar | head -1)
+      - run: docker load < $(ls /tmp/workspace/pack-*-build.tar | head -1)
```

```suggestion
      - run: docker load < $(ls /tmp/workspace/pack-*-run.tar | head -1)
```

Git rebase really doesn't want this fix 🤣

---

lol